fdbcli/TenantCommands.actor.cpp

```diff
@@ -410,7 +410,8 @@ ACTOR Future<bool> tenantListCommand(Reference<IDatabase> db, std::vector<String
 			return false;
 		}
 	}
-	state TenantGroupName tenantGroup(tenantGroupString);
+	state Optional<TenantGroupName> tenantGroup =
+	    tenantGroupString.empty() ? Optional<TenantGroupName>() : Optional<TenantGroupName>(tenantGroupString);
```

Comment:
`tenantGroupString` may need to be an `Optional` too for similar reasons

Reply:
`tenantGroupString` is only there to read from the command line, and gets converted to an `Optional<TenantGroupName>` here

Reply:
My concern is if they specify `tenant_group=""`, which should correspond to an empty tenant group string rather than unset

Reply:
In that scenario, should it return all tenants in the range that have unset tenant groups? Does that mean unset tenant groups are encoded in the `tenantGroupTenantIndex` as empty strings in the tuple too?

Reply:
I think I may be confusing "unset" and "empty". with `unset`, that would be an empty optional, but `empty` means the optional tenant group has the name `""`. Is that right?

Reply:
Yes, the empty string should be considered a valid tenant group from the fdbcli's perspective at least, and an unset tenant group would mean we aren't filtering on groups